include/exadg/acoustic_conservation_equations/spatial_discretization/spatial_operator.h

```diff
@@ -147,12 +149,21 @@ class SpatialOperator : public Interface::SpatialOperator<Number>
   void
   initialize_dof_vector(BlockVectorType & dst) const final;
 
+  void
+  initialize_dof_vector_pressure(VectorType & dst) const;
+
   /*
    * Prescribe initial conditions using a specified analytical/initial solution function.
    */
   void
   prescribe_initial_conditions(BlockVectorType & dst, double const time) const final;
 
+  /*
+   * Set right-hand side vector that has been integrated externally.
```

Comment:
What do you mean by "integrated"? I find this wording quite often in the present PR.

Reply:
It means, that the DoF vector contains 
```math
\int q f \mathrm{d}\Omega
```
where $q$ is an ansatz function and $f$ is the typical RHS "density". If I recall correctly, in other places the RHS is applied from a DoF vector, where the DoF vector stores the RHS density. I decided to use this naming to make this clear but can adapt to another convention. If you agree, I will extend the documentation to ensure this becomes more clear.

Reply:
I would expect a `rhs` vector in ExaDG is always an "integrated" vector. Could you point me to the place where this is used differently in ExaDG?